Sources/Testing/Testing.docc/Attachments.md

```diff
@@ -0,0 +1,32 @@
+# Attachments
+
+<!--
+This source file is part of the Swift.org open source project
+
+Copyright (c) 2024–2025 Apple Inc. and the Swift project authors
+Licensed under Apache License v2.0 with Runtime Library Exception
+
+See https://swift.org/LICENSE.txt for license information
+See https://swift.org/CONTRIBUTORS.txt for Swift project authors
+-->
+
+Attach values to tests to help diagnose issues and gather feedback.
+
+## Overview
+
+Attach values such as strings and files to tests.  Implement the ``Attachable``
```

Comment:
peak nitpick
```suggestion
Attach values such as strings and files to tests. Implement the ``Attachable``
```

Reply:
lolz. Does DocC even care? I'm just going to leave that for now so I don't have to run CI again.